code/modules/mob/living/carbon/human/say.dm

```diff
@@ -257,27 +106,27 @@
 		verb = speaking.get_spoken_verb(ending)
 	else
 		if(ending == "!")
-			verb = "exclaims"
-		else if(ending == "?")
-			verb = "asks"
+			return pick("exclaims", "shouts", "yells")

```

Comment:
If I recall correctly, the old system was broken to hell, and used the pick list anyways.